### PR TITLE
Initial functionality with version command

### PR DESCRIPTION
**Pull request by obiyankenobi**
In this initial code, there's the basic files for a working Ledger app. For now, we can only query the Ledger app for its version.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (jansegre) [inline LICENSE:1]: I think we should use Apache since the original was Apache.

Comment (obiyankenobi) [reply LICENSE:1]: Noticed now that the original project also uses MIT, only uses the Apache license for the Makefile, which was probably copied from another project.

I kept the Apache license on the same file and added a modified notice.

**Review by jansegre — APPROVED** [review]

File: src/getVersion.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#include <stdbool.h>
+#include <os.h>
+#include <os_io_seproxyhal.h>
+#include "hathor.h"
+#include "ux.h"
+
+// handleGetVersion is the entry point for the getVersion command. It
+// unconditionally sends the app version.
+void handleGetVersion(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint16_t dataLength, volatile unsigned int *flags, volatile unsigned int *tx) {
+	G_io_apdu_buffer[0] = APPVERSION[0] - '0';
```
Comment (msbrogli) [inline src/getVersion.c:18]: Maybe we should leave a comment that `APPVERSION = "X.Y.Z"`, where `X`, `Y`, and `Z` must be a single digit between 0 and 9.

**Review by msbrogli — APPROVED** [review]

File: src/getVersion.c (hunk 2 above)
Comment (obiyankenobi) [reply src/getVersion.c:18]: Added this comment on the Makefile, where we define `APPVERSION`.

File: src/getVersion.c [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#include <stdint.h>
+#include <stdbool.h>
+#include <os.h>
+#include <os_io_seproxyhal.h>
+#include "hathor.h"
+#include "ux.h"
+
+// handleGetVersion is the entry point for the getVersion command. It
+// unconditionally sends the app version.
+void handleGetVersion(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint16_t dataLength, volatile unsigned int *flags, volatile unsigned int *tx) {
```
Comment (pedroferreira1) [inline src/getVersion.c:17]: The app magic number should have been sent here also?

In the `getVersion` command we are expecting the version and this magic number.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+$(error Environment variable BOLOS_SDK is not set)
+endif
+include $(BOLOS_SDK)/Makefile.defines
+
+#########
+#  App  #
+#########
+
+APPNAME    = Hathor
+ICONNAME   = nanos_app_hathor.gif
```
Comment (pedroferreira1) [inline Makefile:30]: I think we should change the icon to have our new UI before sending to the "store". Maybe ignore it now if it's not so easy and create an issue to do it before publishing.

(I didn't know how to add a comment on the `.gif` file)

**Review by pedroferreira1 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 4 above)
Comment (obiyankenobi) [reply Makefile:30]: We can do it on a separate PR. We also need to provide a larger icon for the Ledger manager: https://ledger.readthedocs.io/en/latest/additional/publishing_an_app.html#design-guidelines

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**